<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MachineView</class>
 <widget class="QFrame" name="MachineView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QLabel" name="cyanCount">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>83</y>
     <width>80</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #323232;</string>
   </property>
   <property name="text">
    <string>10000/100%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="magentaCount">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>83</y>
     <width>80</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #323232;</string>
   </property>
   <property name="text">
    <string>10000/100%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="yellowCount">
   <property name="geometry">
    <rect>
     <x>428</x>
     <y>83</y>
     <width>80</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #323232;</string>
   </property>
   <property name="text">
    <string>10000/100%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="blackCount">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>83</y>
     <width>80</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #323232;</string>
   </property>
   <property name="text">
    <string>10000/100%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="emergencyStop">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>352</y>
     <width>240</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;
border-radius: 4px;
background-color: #f44336</string>
   </property>
   <property name="text">
    <string>Emergency Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="paperJam">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>352</y>
     <width>240</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;
border-radius: 4px;
background-color: #f44336</string>
   </property>
   <property name="text">
    <string>Invoke Paper Jam</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tempoManage">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>424</y>
     <width>240</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #323232;
border-radius: 4px;
background-color: #f5f5f5</string>
   </property>
   <property name="text">
    <string>Tempo (14400 pph)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deliverManage">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>424</y>
     <width>240</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #323232;
border-radius: 4px;
background-color: #f5f5f5</string>
   </property>
   <property name="text">
    <string>Delivery (17000/100%)</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="feederManage">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>424</y>
     <width>240</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #323232;
border-radius: 4px;
background-color: #f5f5f5</string>
   </property>
   <property name="text">
    <string>Feeder (17000/100%)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cyanManage">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>24</y>
     <width>96</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #323232;
border-radius: 4px;
background-color: #f5f5f5</string>
   </property>
   <property name="text">
    <string>Cyan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="magentaManage">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>24</y>
     <width>96</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #323232;
border-radius: 4px;
background-color: #f5f5f5</string>
   </property>
   <property name="text">
    <string>Magenta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yellowManage">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>24</y>
     <width>96</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #323232;
border-radius: 4px;
background-color: #f5f5f5</string>
   </property>
   <property name="text">
    <string>Yellow</string>
   </property>
  </widget>
  <widget class="QPushButton" name="blackManage">
   <property name="geometry">
    <rect>
     <x>557</x>
     <y>24</y>
     <width>96</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #323232;
border-radius: 4px;
background-color: #f5f5f5</string>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>123</y>
     <width>668</width>
     <height>205</height>
    </rect>
   </property>
   <property name="text">
    <string>SimulatorImage</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>352</y>
     <width>240</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;
border-radius: 4px;
background-color: #00afff</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
